### PR TITLE
Add fs cli support

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: cli/ceph/fs/sub_volume.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+class SubVolume(Cli):
+    """This module provides CLI interface for FS subvolume related operations"""
+
+    def __init__(self, nodes, base_cmd):
+        super(SubVolume, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} subvolume"
+
+    def create(self, vol_name, subvol_name, **kwargs):
```
Comment (vamahaja) [inline cli/ceph/fs/sub_volume.py:12]: better names are `volume`, `subvolume`

Comment (vamahaja) [reply cli/ceph/fs/sub_volume.py:12]: Change for all methods

File: cli/ceph/fs/sub_volume.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            subvol_name (str): Name of the subvol
+            kw: Key/value pairs of configuration information to be used in the test.
+        """
+        cmd = f"{self.base_cmd} create {vol_name} {subvol_name} {build_cmd_from_args(**kwargs)}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def rm(self, vol_name, subvol_name, group_name=None, force=False):
```
Comment (vamahaja) [inline cli/ceph/fs/sub_volume.py:26]: `group`

Comment (vamahaja) [reply cli/ceph/fs/sub_volume.py:26]: Change for all methods

**Review by vamahaja — CHANGES_REQUESTED** [review]

**Comment by pranavprakash20** [comment]
changed the variable names

**Review by MohitBis — DISMISSED** [review]
LGTM

File: cli/ceph/fs/sub_volume.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    def ls(self, volume, group):
+        """
+        List subvol groups
+        Args:
+            volume (str): Name of vol where subvol is present
+            group (str): subvol group name
+        """
+        cmd = f"{self.base_cmd} ls {volume}"
+        if group:
```
Comment (vamahaja) [inline cli/ceph/fs/sub_volume.py:53]: `group` is mandatory parameter

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: cli/ceph/fs/sub_volume.py (hunk 3 above)
Comment (pranavprakash20) [reply cli/ceph/fs/sub_volume.py:53]: added group as optional param

**Comment by pranavprakash20** [comment]
@vamahaja review comment addressed

**Review by vamahaja — APPROVED** [review]

**Pull request merged**